lib/cmd/fh3/projects.js

```diff
@@ -112,6 +112,15 @@ function createProject(args, deployEnvironment, cb) {
     });
   }
 
+  // Like NGUI, filter out appTemplates which are
+  // not 'selected' before calling create
+  function filterSelectedApps(template) {
+    var filteredTemplate = _.clone(template);
```

Comment:
+1, if template isn't used elsewhere (after mutation) as you mention, the clone here is unnecessary. Will fix up.